From 4ce3545b704588c5889b8dd7b100fcdb88ae2e1a Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Wed, 17 Sep 2014 17:48:54 +0100
Subject: [PATCH] iprutils: Don't use gettext

... since we don't have any translation.

Based on the former patch by Jeremy Kerr.

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
[Thomas: adapt to the new autotools build system.]
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 configure.ac | 2 +-
 iprconfig.h  | 4 +---
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index ee69bee..c1a4f70 100644
--- a/configure.ac
+++ b/configure.ac
@@ -137,7 +137,7 @@ AC_SUBST([IPRCONFIG_LIBS])
 LIBS="$save_LIBS"
 
 # Checks for header files.
-AC_CHECK_HEADERS([fcntl.h libintl.h locale.h netinet/in.h nl_types.h \
+AC_CHECK_HEADERS([fcntl.h locale.h netinet/in.h nl_types.h \
 		 paths.h stddef.h stdint.h stdlib.h string.h sys/file.h \
 		 sys/ioctl.h sys/mount.h sys/socket.h syslog.h \
 		 unistd.h ncurses.h form.h menu.h], [],
diff --git a/iprconfig.h b/iprconfig.h
index 9bce950..751a737 100644
--- a/iprconfig.h
+++ b/iprconfig.h
@@ -11,9 +11,7 @@
  *
  **/
 
-#include <libintl.h>
-
-#define _(string) gettext(string)
+#define _(string) (string)
 #define __(string) (string)
 #define EXIT_FLAG		0x8000	/* stops at given screen on exit call */
 #define CANCEL_FLAG		0x4000	/* stops at given screen on quit call */
-- 
2.6.4

